public/app/plugins/datasource/elasticsearch/query_builder.ts

```diff
@@ -419,4 +420,41 @@ export class ElasticQueryBuilder {
       aggs: this.build(target, null, querystring).aggs,
     };
   }
+
+  addPPLAdhocFilters(queryString: any, adhocFilters: any) {
```

Comment:
seems this function do nothing, do we need it?

Reply:
Ad hoc filters are implemented in this PR: https://github.com/open-o11y/grafana/pull/10